demo/run.sh

```diff
@@ -60,30 +59,43 @@ function createHostFactoryToken() {
     client \
     /conjurinc/ansible/create_host_token.sh
 
-  echo '-----'
+  export HFTOKEN=$(<conjur-client-files/output/hftoken.txt)
```

Comment:
I know this is a demo script, but we shouldn't `export` secrets at the end of bash functions. That leaks them all over the environment, hard to say what will pick it up.